tests/vmlifecycle_test.go

```diff
@@ -41,6 +42,15 @@ import (
 
 var _ = Describe("Vmlifecycle", func() {
 
+	primaryNodeName := os.Getenv("primary_node_name")
+	if primaryNodeName == "" {
+		primaryNodeName = "master"
+	}
+	dockerTag := os.Getenv("docker_tag")
+	if dockerTag == "" {
+		dockerTag = "devel"
```

Comment:
I think it should be "latest" as default, to be consistent with the rest. Everywhere else we have latest as default, and explicitly set devel in the scripts.